posthog/session_recordings/session_recording_api.py

```diff
@@ -322,9 +338,19 @@ def safely_get_object(self, queryset) -> SessionRecording:
         return recording
 
     def list(self, request: request.Request, *args: Any, **kwargs: Any) -> Response:
-        filter = SessionRecordingsFilter(request=request, team=self.team)
-        self._maybe_report_recording_list_filters_changed(request, team=self.team)
-        return list_recordings_response(filter, request, self.get_serializer_context())
+        use_query_type = (request.GET.get("as_query", "False")).lower() == "true"
```

Comment:
let's the API client declare which processing version it accepts so we can test the new mechanism with a slow rollout